### PR TITLE
Add 'Hamburger Sparkasse AG' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22haspa%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22categories%22%3A%5B%22finance%22%5D%2C%22name%22%3A%22Hamburger%20Sparkasse%20AG%22%2C%22address%22%3A%22Ecke%20Adolphsplatz%20%2F%20Gr.%20Burstah%5Cn20457%20Hamburg%22%2C%22email%22%3A%22haspa%40haspa.de%22%2C%22web%22%3A%22https%3A%2F%2Fhaspa.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.haspa.de%2Fde%2Fhome%2Ffooter%2Fimpressum.html%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1304%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

File: companies/haspa.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Hamburger Sparkasse AG",
+    "address": "Ecke Adolphsplatz / Gr. Burstah\n20457 Hamburg",
+    "email": "haspa@haspa.de",
+    "web": "https://haspa.de",
```
Comment (moriarty-jim) [inline companies/haspa.json:12]:
```suggestion
    "web": "https://haspa.de",
    "phone": "+49 40 35790",
    "fax": "+49 40 35793418",
```
added generic phone & fax

File: companies/haspa.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "sources": [
+        "https://www.haspa.de/de/home/footer/impressum.html",
```
Comment (moriarty-jim) [inline companies/haspa.json:14]:
```suggestion
        "https://www.haspa.de/de/home/footer/impressum.html",
        "https://www.haspa.de/de/home/footer/datenschutz.html",
```
added privacy page

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
changes to address, email; added phone, fax, source

File: companies/haspa.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "slug": "haspa",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "Hamburger Sparkasse AG",
+    "address": "Ecke Adolphsplatz / Gr. Burstah\n20457 Hamburg",
```
[the remark this answers is not in the document]
Comment (baltpeter) [reply companies/haspa.json:10]: According to https://www.postdirekt.de/plzserver/, 20454 is a "Großempfängerpostleitzahl", so the "Postfach" shouldn't be necessary despite what the privacy policy says.

```suggestion
    "address": "20454 Hamburg\nDeutschland",
```

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**